web/gui/dashboard_info.js

```diff
@@ -1099,67 +1099,86 @@ netdataDashboard.context = {
     },
 
     'apps.file_open': {
-        height: 2.0
+        height: 2.0,
```

Comment:
Do you really need to double the height of the charts?

Reply:
You are right, I tested now and I liked the final result. I removed it. Thank you!